docs/articles/documentation/using-testcafe/programming-interface/livemoderunner.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+title: LiveModeRunner Class
+permalink: /documentation/using-testcafe/programming-interface/livemoderunner.html
+---
+# LiveModeRunner Class
+
+An object that configures and launches test tasks in [live mode](../common-concepts/live-mode.md). In this mode, TestCafe watches for changes you make in the test files. These changes immediately restart the tests so that you can see the effect.
```

Comment:
An object that configures and launches test tasks in [live mode](../common-concepts/live-mode.md). In this mode, TestCafe watches for changes you make in the test files, restarts the tests, and reflects the changes.

Reply:
I'm not sure about 'reflects'. TestCafe runs tests, so that you can look at the browser (not TestCafe) and see the changes. Like, TestCafe doesn't directly 'reflect' changes. Or you think it's okay?

Reply:
Perhaps 'applies'? An object that configures and launches test tasks in live mode. In this mode, TestCafe watches for changes you make in the test files, restarts the tests, and applies the changes.

Reply:
'Applies' is even more active. It would mean that TestCafe does something with these changes. While it actually just re-runs tests when it detects that changes were made.

Okay, since this is not a dedicated topic about live mode, this sentence is fine without the last part:

*An object that configures and launches test tasks in live mode. In this mode, TestCafe watches the test files, and restarts the tests when you make changes.*

Reply:
Ok